### ISSUE TITLE
Rolling Favorites: remove Kate, Dolphin and Firefox

### ISSUE THREAD

**Issue by star-buck** [blue-systems/netrunner-releases#188]
![Screenshot_20190410_142022](https://user-images.githubusercontent.com/840070/55878023-d8fc9080-5b9b-11e9-8510-5ed2200c0ca6.png)

**Comment by NuLogicSystems** [comment]
We only have set to be there "org.kde.ksysguard.desktop", "systemsettings.desktop", "org.kde.discover", "octopi.desktop". These others are added by the system itself after. I have yet to figure out how to keep them from being added, and/or how to remove them. This has actually been bothering me for a while now. Same goes for initially frequently used on the first page of system settings.

**Comment by llelectronics** [comment]
In the LNF you can add scripting for all default menus. Here you can set the default favorites. It should have priority over everything the system ships by default.

**Comment by NuLogicSystems** [comment]
They are, and are that way at first, but for some reason the system adds the others back in.
https://github.com/netrunner-rolling/artwork-lnf-theme-netrunner-black/blob/master/usr/share/plasma/look-and-feel/org.kde.netrunner-black.desktop/contents/plasmoidsetupscripts/org.kde.plasma.kickerdash.js

If you remember correctly, I'm the one that found that this scripting could be done in the LNF to begin with.

**Comment by star-buck** [comment]
ah okay... i switched to simplemenu to take screenshots, thats why they appeared again in all menues... that still shouldnt be?

**Comment by NuLogicSystems** [comment]
Yea, tell me about it. It's been anointing me for a long time now. I also prefer simplemenu.

**Comment by star-buck** [comment]
@Pointedstick : you know if that is a bug or what we might be missing here?

**Comment by llelectronics** [comment]
Can you make sure that the LNF is really used in the system that boots. Otherwise it could be a bug aswell.

**Comment by NuLogicSystems** [comment]
The lnf is used when first applied, and only our favorites are added. It's only after changing the containment  (menu alternative) that it adds both our favorites from the lnf, and also the upstream defaults.

**Comment by star-buck** [comment]
Can we remove the upstream defaults somewhere?

**Comment by Pointedstick** [comment]
> @Pointedstick : you know if that is a bug or what we might be missing here?

I'm not familiar with the issue, sorry.

**Comment by star-buck** [comment]
Then it seems like a new issue.
